pkg/controller/sparkapplication/controller.go

```diff
@@ -672,9 +683,54 @@ func (c *Controller) submitSparkApplication(app *v1beta1.SparkApplication) *v1be
 			}
 		}
 	}
+
+	//Use PodGroup to schedule spark application executor pods if required
+	if c.volcanoClient != nil && app.VolcanoSchedulingRequired() {
+		if app.Spec.Executor.Annotations == nil {
+			app.Spec.Executor.Annotations = make(map[string]string)
+		}
+		if _, ok := app.Spec.Executor.Annotations["scheduling.k8s.io/group-name"]; !ok {
```

Comment:
Please define `scheduling.k8s.io/group-name` as a constant.

Reply:
we already have this const in volcano, please help to reuse that const :)